crates/repl/src/repl_store.rs

```diff
@@ -22,6 +22,8 @@ pub struct ReplStore {
     enabled: bool,
     sessions: HashMap<EntityId, View<Session>>,
     kernel_specifications: Vec<KernelSpecification>,
```

Comment:
Now that this list is _just_ the global Jupyter kernels, we should probably name it as such.